### PR TITLE
Added support for Stack Overflow, white spaces on some files

### PR DESCRIPTION
**Pull request by chivoX**
happy hacktoberfest ! 

Adding support for Stack Overflow !

### CODE REVIEW

File: lib/jobby_job_job/posting_processor.rb [hunk 1]
```diff
@@ -27,6 +27,11 @@ def initialize
           url:"https://remoteok.io/remote-dev-jobs.json",
           format:"json",
           parser:"RemoteOk"
+        },
+        {
+          url:"https://stackoverflow.com/jobs/feed",
```
Comment (willywos) [inline lib/jobby_job_job/posting_processor.rb]: Is there a way to set the location to "Remote" so it only returns remote jobs?

Comment (willywos) [reply lib/jobby_job_job/posting_processor.rb]: Yeah like on Indeed, you can put "Remote" into the location field and it should return Remote jobs. I think you can do that on Stack Overflow.

Comment (chivoX) [reply lib/jobby_job_job/posting_processor.rb]: ooooooh, i see what you mean..... ok let me take a look at it.

do you think you can maybe marge this first and create a new issue to add the remote filter :D ?
that would be +2 towards the shirt ! haahha

Comment (willywos) [reply lib/jobby_job_job/posting_processor.rb]: haha yeah I can do that for you. Cheers🍻

**Pull request merged**